testing/src/test/kotlin/com/airbnb/android/mvrx/test/TestRuleView.kt

```diff
@@ -0,0 +1,21 @@
+package com.airbnb.android.mvrx.test
+
+import com.airbnb.mvrx.BaseMvRxFragment
+
+class TestRuleView : BaseMvRxFragment() {
+
+    var subscribeCallCount = 0
+
+    private val viewModel = TestRuleViewModel()
+
+    init {
+        viewModel.subscribe {
+            subscribeCallCount++
```

Comment:
This is actually a fragment, I think the fact that it is in `init` is confusing, but I understand that's probably easier than driving lifecycle in the tests.

Reply:
So, this was the exact problem. This subscribe is an extension function available inside MvRxView. Without the TestRule, this callback was never called in unit tests.

Reply:
Exactly, this is a view and it would not be called before. There is a test `MvRxTestRuleTestLifecycleAwareObserverEnabled` where you can see that this wouldn't be called. The reason I used `init` is because I was trying to be consistent and use same approach that you guys have in `TestRuleViewModel`

Reply:
Got it. Thanks for the explanation